### PR TITLE
Use default preset label if preset is not set

### LINKED ISSUE #149: (undefined) message displayed on the dashboard
Capture d’écran 2023-06-06 à 16 08 24

Not sure what it means in this context, but it is certainly disturbing in term of ux

### PR DESCRIPTION
**Pull request by evidolob**
Fix #149

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -255,15 +255,6 @@ async function initializeCrc(
   return hasSetupFinished;
 }
 
-// function initCommandsAndPreferences(
```
Comment (gbraad) [inline src/extension.ts:258]: do not leave code that has been commented. in not sued, get rid of it. we have Git to handle this.

File: src/extension.ts [hunk 2]
```diff
@@ -255,15 +255,6 @@ async function initializeCrc(
...
-//   provider: extensionApi.Provider,
-//   extensionContext: extensionApi.ExtensionContext,
-//   telemetryLogger: extensionApi.TelemetryLogger,
-// ): void {
-//   addCommands(telemetryLogger);
-//   syncPreferences(provider, extensionContext, telemetryLogger);
```
Comment (gbraad) [inline src/extension.ts:264]: not needed?

**Comment by gbraad** [comment]
> requested review from [gbraad](https://github.com/gbraad) [1 hour ago](https://github.com/crc-org/crc-extension/pull/163#event-9820702924)

but nothing has changed since

**Comment by evidolob** [comment]
But what I should change? Delete commended code? (Which I'm doing already)

**Comment by gbraad** [comment]
So this code was not needed?

**Comment by evidolob** [comment]
Yes, I comment and commit that code a while ago, and in this PR just remove it.

**Comment by gbraad** [comment]
I was wondering how this got in there before. I am normally quite precise about committing commented code.

Can also verify this on Monday.

**Review by gbraad — APPROVED** [review]

**Pull request merged**